static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-support-baseline.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+import { makeABTest } from 'common/modules/commercial/contributions-utilities';
+import template from 'lodash/utilities/template';
+import singleButtonTemplate from 'raw-loader!common/views/acquisitions-epic-single-button.html';
```

Comment:
This could really be an inlined template literal. We could ditch the `lodash/utilities/template` then :)